src/backend/metal/src/internal.rs

```diff
@@ -451,7 +451,10 @@ pub struct ServicePipes {
 
 impl ServicePipes {
     pub fn new(device: &metal::DeviceRef) -> Self {
+        #[cfg(target_os = "macos")]
         let data = include_bytes!("./../shaders/gfx_shaders.metallib");
```

Comment:
let's rename the old one for consistency to `gfx_shaders-macos.metallib`.
Perhaps, we could use the dashes in both even? i.e. `gfx-shader-macos.metallib`